core/arch/arch_exports.h

```diff
@@ -62,6 +62,7 @@
 #    define OPMASK_REG_SIZE 8
 #    define MCXT_SIMD_SLOT_SIZE ZMM_REG_SIZE
 #    define MCXT_TOTAL_SIMD_SLOTS_SIZE (MCXT_NUM_SIMD_SLOTS * MCXT_SIMD_SLOT_SIZE)
+#    define MCXT_TOTAL_OPMASK_SLOTS_SIZE (MCXT_NUM_OPMASK_SLOTS * OPMASK_REG_SIZE)
```

Comment:
Please update the commit description to explain that actually filling these values in will be in a future patch.

Reply:
Ok, comment added.